### ISSUE TITLE
[NRRPLT-8729] Implement a Experiment Manager Layer in NRPCore Python Client

### ISSUE THREAD

**Issue by EloyRC** [EloyRC/nrp-core#63]
<p>The current implementation of the NRPCore Python Client is designed to handle a single experiment. This means that when running multiple experiment in parallel users need to take care of advancing each experiment one by one, to check for exceptions, etc.</p>

<p>It would be nice to have an Experiment Manager ontop of the current python client which takes cares of conveniently handling multiple instances of NRPCore. The design details of this manager are to be decided, for example from actual needs discovered it ongoing attempts to run distributed experiments in nrp-core, e.g. <a href="https://hbpneurorobotics.atlassian.net/browse/NRRPLT-8498" title="smart-link" class="external-link" rel="nofollow noreferrer">https://hbpneurorobotics.atlassian.net/browse/NRRPLT-8498</a></p> <p><b>Created:</b> Fri, 23 Dec 2022 12:05:06 +0100</p>
